src/mainboard/clevo/adl-p/devtree.c

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <chip.h>
+#include <cpu/intel/turbo.h>
+#include <device/device.h>
+#include <device/pci_def.h>
+#include <option.h>
+#include <types.h>
+#include <variants.h>
+
+void devtree_update(void)
+{
+	config_t *cfg = config_of_soc();
+
+	// read the CpuThrottlingThreshold efivar
+	uint8_t CpuThrottlingThreshold;
+	CpuThrottlingThreshold = get_cpu_throttling_threshold();
+
+
+	// read TjMax
```

Comment:
This can probably be defined statically as a Kconfig option for each CPU generation, I think.